### DISCUSSION TITLE
restart the computer ，chia-blockchain is 1.1.1, but can't find the plot which had ploted before

### DISCUSSION THREAD

**Discussion by wenbsu** [Chia-Network/chia-blockchain#2446] [Q&A]
When I rebooted my computer and opened my wallet again, I found that all the work I had done was useless 
The previous drawing is invalid
![image](https://raw.githubusercontent.com/wenbsu/mypicture/main/addDirectory.bmp)
![image](https://raw.githubusercontent.com/wenbsu/mypicture/main/invalidation.bmp)
![image](https://raw.githubusercontent.com/wenbsu/mypicture/main/previewDirectory.bmp)

**Comment by leoliyuanduo** [comment]
你用的是v1.05吗？ 你downgrade到v1.05再试试看能不能farm
